graylog2-web-interface/src/components/widgets/Widget.jsx

```diff
@@ -204,37 +207,34 @@ const Widget = React.createClass({
       return <span />;
     }
     const showConfigModal = (
-      <WidgetConfigModal ref="configModal"
-                         dashboardId={this.props.dashboardId}
+      <WidgetConfigModal dashboardId={this.props.dashboardId}
                          widget={this.props.widget}
                          boundToStream={this._isBoundToStream()} />
     );
 
     const editConfigModal = (
-      <WidgetEditConfigModal ref="editModal"
-                             widget={this.props.widget}
+      <WidgetEditConfigModal widget={this.props.widget}
```

Comment:
We still use the `editModal` ref to open the edit modal dialog, so it can't be removed.